docs/content/access-api-rate-limits.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Flow Access API rate limits on Dapper Labs Access nodes
+---
+
+#### Rate limits
+
+Following are the current rate limits for the [Access Node gRPC API](/access-api).
+Once the limit has reached, the client will receive an RPC error `ResourceExhausted` in the gRPC response.
+
+Please note, these limits only apply to the access nodes hosted by Dapper Labs. Access nodes run by other node operators will have different rate limits.
+
+
+##### Mainnet
+
+|                 gRPC API                             | Total request per second per client IP        |
+|:-----------------------------------------------------|:---------------------------------------------:|
+| flow.access.AccessAPI/Ping                           |                   2000                        |
```

Comment:
Given that the whole document is for access node APIs, maybe remove the `flow.access.AccessAPI/` prefix, it is always the same and implied

Reply:
so i wanted to differentiate this with the legacy API which I think has a different prefix.